soc/arm/bcm_vk/viper/dts_fixup.h

```diff
@@ -0,0 +1,13 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/*
+ * Copyright 2020 Broadcom.
+ */
+
+/* SoC level DTS fixup file */
+
+#if defined(CONFIG_SOC_BCM58402_M7)
```

Comment:
These have been phased out, you shouldn't need to set these or have dts_fixup.h at all.

Reply:
ACK. Will remove dts_fixup.h